configs/staff_hours.yaml

```diff
@@ -27,11 +27,20 @@ staff-hours:
   Monday:
   Tuesday:
   Wednesday:
+   - time: ['18:00', '20:00']
```

Comment:
We close at 6pm on Wednesdays fwiw.

Also holy shit thank you for adding so many hours